pkgs/top-level/all-packages.nix

```diff
@@ -27822,6 +27822,8 @@ with pkgs;
 
   calibre-web = callPackage ../servers/calibre-web { };
 
+  inherit (callPackage ../applications/networking/cluster/calico {}) calico-apiserver calico-app-policy calico-cni-plugin calico-kube-controllers calico-pod2daemon calico-typha calicoctl confd-calico;
```

Comment:
This line is quite long, it would look better if we split these.

Reply:
fair, but I am unclear on how to break apart these words, can you make a proposal that is syntactically valid?

Reply:
Nix is not whitespace or indentation bound, You can do something like this:
```nix
  inherit (callPackage ../applications/networking/cluster/calico {})
    calico-apiserver
    calico-app-policy
    calico-cni-plugin
    calico-kube-controllers
    calico-pod2daemon
    calico-typha
    calicoctl
    confd-calico;
```
Or you can just split however you like while keeping columns 80 character wide. Nixpkgs doesn't have a well-defined coding standard yet... It just looks more readable like this IMO.